modules/client/main.tf

```diff
@@ -125,6 +125,15 @@ resource "google_compute_instance" "forseti-client" {
     scopes = ["cloud-platform"]
   }
 
+  dynamic "shielded_instance_config" {
+    for_each = var.client_shielded_instance_config == null ? [] : [var.client_shielded_instance_config]
+    content {
+      enable_secure_boot          = lookup(var.client_shielded_instance_config, "enable_secure_boot", null)
```

Comment:
Not sure what's the convention, do we usually default it to null or false for bools?

Reply:
What I found out is.... if I specify a shielded image without specifying any of these shielded properties, by default, ` vTPM` and `Integrity Monitoring`  are set to `true` and `Secure Boot` is set to `false`. Because of that, I feel it's misleading to set these properties to `false` if missing. Setting them to `null` allows GCP to define the correct default values for them.

Also, I see the same conventions being used in other CFT modules. So, I try  to follow the same pattern  here.

Reply:
Sounds good. Thanks for looking into this.